apps/parse-rated-paragraphs/src/app/parse-rated.ts

```diff
@@ -0,0 +1,52 @@
+import * as fs from 'fs';
+import * as util from 'util';
+import {
+  ParsedArticleParagraphStandalone,
+  ArticleParagraphBacksUpClaim,
+} from '@foodmedicine/interfaces';
+
+const allParagraphsBasePath = './tmp/correlated-paragraphs/';
+const readFileAsync = util.promisify(fs.readFile);
```

Comment:
Nit: typically global constants like these are defined in capital snake case:
`const ALL_PARAGRAPHS_BASE_PATH = '...';`

Reply:
Edit: this is more applicable when the value is a primitive, so this may not apply to `readFileAsync`